kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -448,6 +477,41 @@ protected static void ensureValidMap(String name, List<String> values) {
     return patternList;
   }
 
+  public String getSingleMatch(String value,
```

Comment:
A non-static public getSingleMatch() inside `BigQuerySinkConfig` is a bit confusing. Suggest put it back to `TopicToTableResolver`

Reply:
If arguments listed in https://github.com/wepay/kafka-connect-bigquery/pull/238#discussion_r372871337 won't convince you, I'll do it without any further discussions 👍

Reply:
Let's keep this function here. Besides, can you please
1) add doc
2) adjust function param alignment
3) check if it is possible to reuse `getSingleMatch()` in `getSingleMatches()`

Reply:
It wasn't possible to reuse getSingleMatch in getSingleMatches because only one of them performs substitution of captured RegEx groups, but I was able to extract the common part of looking up the matching pattern.